app/src/main/res/values/strings.xml

```diff
@@ -38,10 +38,9 @@
   <string name="topic_play_chapter_index">%s.</string>
   <string name="chapter_name">Chapter %s: %s</string>
   <string name="chapter_completed">Chapter %s: %s is completed.</string>
-  <string name="fractions_hint_text">Write fraction here.</string>
-  <string name="number_input_hint_text">Write numbers here.</string>
+  <string name="fractions_default_hint_text">Enter a fraction in the form x/x, or a mixed number in the form x x/x.</string>
```

Comment:
I was using the error message as same as that of web ...PTAL on  #419

Reply:
Sorry, I'm not sure I see how #419 relates to this PR. This PR is meant to change the non-error hint text to match Oppia web.